### ISSUE TITLE
[Public Catalog] Implement I/O Banner

### ISSUE THREAD

**Issue by jpvajda** [newrelic/developer-website#1570]
Marketing will provide a banner for I/O that will exist on the main catalog page. It's been discussed that the banner will be the size of the NPM banner currently on the home page of Docs with a CTA button that will open a video introducing the I/O concept. This video would play in a small video component that would open on click of the CTA.

Note: This could be animated.

### Acceptance Criteria 

- [x] obtain design from marketing https://github.com/newrelic/developer-website/issues/1414
- [x] evaluate scope 
- [ ] obtain logo SVG and final Banner elements from Moe Hung eta Sept 9 (use place holder until we have it)
- [ ] implement banner. the banner would be the same tone in light mode as dark mode.
- [x] The banner should align with the Search Bar field so the edges of the banner match the edges of the field (note: the mock up is showing the banner exceeding the width of the search field)
- [x] for mobile: scale the image in the flex box so it sizes accordingly (see [docs.newrelic.com banner)](https://github.com/newrelic/docs-website/blob/253610dca7f6d2208ee6dd5a6df55f744e231caa/src/pages/index.js#L30)
- [x] implement CTA button
- [x] Banner CTA button should be instrument on click. 
- [ ] implement video component to play video. the default YouTube frame should be `854` by `480` (the video is hosted on YouTube)
- [x] [mvp] open video in a new tab on youtube 
- [ ] [stretch] open video in a lightbox per the mockup

### Mock Up 🧑‍🎨 

![banner](https://user-images.githubusercontent.com/1395158/131733478-765685e8-c58c-4201-a44c-22ea777fe39b.png)

![video](https://user-images.githubusercontent.com/1395158/131737464-0c4e3c92-a426-44a8-840f-4c1e97ed71bd.png)

**Comment by jpvajda** [comment]
This is blocked on design until next week.

**Comment by jpvajda** [comment]
We are unblocked now, trying to get that world logo svg from marketing

**Comment by jpvajda** [comment]
The lightbox is making this really complex so we'll add it as a stretch goal if we have time

**Comment by jpvajda** [comment]
Confirmed: Banner should be available on Sept 10
